mm2src/mm2_main/src/lp_ordermatch/best_orders.rs

```diff
@@ -260,6 +260,16 @@ pub async fn best_orders_rpc(ctx: MmArc, req: Json) -> Result<Response<Vec<u8>>,
                     BestOrdersAction::Buy => order.as_rpc_best_orders_buy(address, conf_settings, false),
                     BestOrdersAction::Sell => order.as_rpc_best_orders_sell(address, conf_settings, false),
                 };
+                if let Some(original_tickers) = ordermatch_ctx.original_tickers.get(&coin) {
+                    for ticker in original_tickers.iter() {
```

Comment:
I believe there is no need for calling `iter()`

Reply:
Fixed, thanks 🙂